hale-docker.conf

```diff
@@ -0,0 +1,3 @@
+global {
```

Comment:
The file should be created in the GitHub Actions runner as part of the workflow.

Here is an example from another workflow that creates a text file:

https://github.com/wetransform/gha-workflows/blob/d110e2ff0307f9718d2e122075de9320018bb831/.github/workflows/sbt-library.yml#L52-L57